src/components/app.jsx

```diff
@@ -25,24 +25,20 @@ const App = React.createClass({
           <div className="container" id="content" style={{marginBottom: 100}}>
             <section className="jumbotron" style={{marginTop: "1em"}}>
               <div className="row">
-                <div className="col-lg-3">
-                  <p>
-                    <img
-                      alt="Logo OpenFisca"
-                      className="img-responsive"
-                      src={url.resolve(config.websiteUrl, "/hotlinks/logo-openfisca.svg")}
-                    />
-                  </p>
+                <div className="col-md-3">
+                  <img
+                    alt="Logo OpenFisca"
```

Comment:
> you repeat “OpenFisca” in the sentence right after it (which might not be so useful, btw)

I tried and it's quite strange for the sentence to start by "référence ...".

I propose to keep the logo, use `alt="OpenFisca"` and display the sentence "1468 variables et 1844 paramètres modélisent le système socio-fiscal français.".